pftools/python/parflow/tools/core.py

```diff
@@ -231,6 +231,7 @@ class Run(BaseRun):
     """
     def __init__(self, name, basescript=None):
         super().__init__(None)
+        self._accessor = None
```

Comment:
Should this be named `_accessor_` with the extra underscore at the end so it is considered to be a private variable [here](https://github.com/parflow/parflow/blob/8f8703d20daafdf56f2d3082cd389886db3e00f1/pftools/python/parflow/tools/database/core.py#L67-L72) for the check [here](https://github.com/parflow/parflow/blob/8f8703d20daafdf56f2d3082cd389886db3e00f1/pftools/python/parflow/tools/database/core.py#L203)?

Reply:
Good catch... Thanks for finding that.